src/acl/acl/edx.py

```diff
@@ -0,0 +1,35 @@
+"""EDX database client."""
+import logging
+
+from email_validator import EmailNotValidError, validate_email
+
+from .exceptions import EdxException
+
+logger = logging.getLogger(__name__)
+
+
+async def user_courses(database, email):
+    """Get logged user courses."""
+
+    if email is None:
+        raise EdxException("An email is required to get user courses")
+
+    # Validate input email so that no SQL injection attack can be made
+    try:
+        valid = validate_email(email)
+    except EmailNotValidError as error:
+        raise EdxException(f"Invalid input email {email}") from error
```

Comment:
It's a very tiny detail...
```suggestion
        raise EdxException(f"Invalid input email: {email}") from error
```